testcases/OpTestKexec.py

```diff
@@ -56,13 +56,13 @@ def setUp(self):
         self.cv_SYSTEM.goto_state(OpSystemState.OS)
         res = self.c.run_command("cat /etc/os-release")
         if "Ubuntu" in (res[0] or res[1]):
-            self.distro = "ubuntu"
+            self.distro = "Ubuntu"
         elif 'Red Hat' in (res[0] or res[1]):
-            self.distro = 'rhel'
+            self.distro = 'RHEL'
         elif 'SLES' in (res[0] or res[1]):
-            self.distro = 'sles'
+            self.distro = 'SLES'
         else:
-            raise self.skipTest("Test currently supported only on ubuntu, sles and rhel")
+            raise self.skipTest("Test currently supported only on Ubuntu SLES and RHEL")
```

Comment:
My previous comment on Distro names was only meant for this line. I see you have changed the names in if {} else {} section as well. Since self.distro is not used anywhere I guess changing to. uppercase should be okay. BTW in the above line , is missing after Ubuntu.

Also can you fold this change into the previous patch and submit a single patch?

Rest of the changes looks good to me.

Reply:
@sacsant Thanks for your review comments